servers/src/main/java/tachyon/worker/package-info.java

```diff
@@ -13,14 +13,6 @@
  * The thrift service is mostly responsible for metadata operations (with a few operations that
  * effect the worker's cached memory).
  *
- * <h3>Checkpoint</h3>
```

Comment:
we removed this functionality from the worker with https://github.com/amplab/tachyon/pull/1890, right?